lib/import.js

```diff
@@ -1,3 +1,5 @@
+'use strict';
```

Comment:
Always use strict. This is the kind of stuff that should be caught by a linter https://github.com/CartoDB/cartodb-nodejs/issues/39